example/provider-local/managedseed.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: seedmanagement.gardener.cloud/v1alpha1
```

Comment:
I propose to add both `managedseed.yaml` and `shoot-managedseed.yaml` into a dedicated directory `managedseed` to make it clear they belong to the same scenario and are supposed to be used together.